v5/cli/conductor.rst

```diff
@@ -1,3 +1,79 @@
-===============
-Conductor (CLI)
-===============
+====================
+Conductor User Guide
+====================
+
+Introduction
+============
+
+Conductor is PROS's templating and project management facility. It is
+responsible for resolving versions of the PROS kernel, user libraries. It also
+provides information about a PROS project to other PROS facilities, like
+providing the target platform (e.g. V5 or Cortex) to the upload utility.
+
+There are a few concepts central to Conductor's functionality:
+
+- Templates: Templates are a description of a set of files. They have a name,
+  version, supported target, and possibly other metadata. Templates typically
+  come in one of 3 forms: remote, local, and installed. Remote templates are
+  templates that can be downloaded. Local templates are a downloaded template
+  that can be installed to a project, even when you're offline. An installed
+  template is a template which has been copied into a PROS project.
+- Depots: A depot is a connector to a remote template storage. Conductor doesn't
+  require any particular access method to these remote templates, but they are
+  typically available from a download link via HTTP. It is possible to implement
+  an SCP, FTP, or other sorts of file transfer depots. The default depot is called
+  pros-mainline and always points to the official repository of templates.

```

Comment:
This could be clarified I think. Currently, it reads as if all templates are in mainline, and mainline points to each template's official repository (instead of mainline being the official repo)